modules/grpc/otel/otel-protobuf-parser.cpp

```diff
@@ -1091,10 +1110,7 @@ syslogng::grpc::otel::ProtobufParser::store_raw_metadata(LogMessage *msg, const
 {
   std::string serialized;
 
-  /* HOST */
-  std::string hostname = _extract_hostname(peer);
-  if (hostname.length())
-    log_msg_set_value(msg, LM_V_HOST, hostname.c_str(), hostname.length());
+  msg->saddr = _extract_saddr(peer);
```

Comment:
After this change the `$HOST` macro will always be empty. Isn't this a problem? Shouldn't we store this value in `$HOST` as well?